### PR TITLE
Document how to roll back

### PR DESCRIPTION
**Pull request by marccarre**
This aims to improve the documentation following a question from an user, as I couldn't see anything in this repository, nor on [weave.works/docs](https://www.weave.works/docs/).

### CODE REVIEW

File: site/using.md [hunk 1]
```diff
@@ -151,6 +151,28 @@ default:deployment/helloworld  helloworld  quay.io/weaveworks/helloworld:master-
...
+- [`deautomate`](#turning-off-automation) to prevent Flux from automatically updating to newer versions, and
+- [`release`](#releasing-a-controller) to deploy the version you want to roll back to.
+
+```sh
+$ fluxctl deautomate --controller=default:deployment/helloworld
+Commit pushed: c07f317
+CONTROLLER                     STATUS   UPDATES
+default:deployment/helloworld  success
+
+$ fluxctl release --controller=default:deployment/helloworld --update-image=quay.io/weaveworks/helloworld:master-a000001
```
Comment (squaremo) [inline site/using.md]: Might be worth also putting `fluxctl list-images` in here, to show which images are available to be released.

Comment (marccarre) [reply site/using.md]: Sure, maybe even `list-images` before and after, to make it easier to visualise the impact?

Comment (marccarre) [reply site/using.md]: Done.

**Comment by squaremo** [comment]
You may click the big friendly merge button @marccarre :)

**Pull request merged**